client/src/components/RouteDisplay/RouteSummary/RouteSummaryItem.scss

```diff
@@ -0,0 +1,21 @@
+@import "src/css/Common";
+
+.row-summary-item {
```

Comment:
Do you intend to name it like this or just wrong tpying ?

Reply:
good catch, fixed

Reply:
I always name the css after the component name. So this one is a mistake.

Reply:
Okayy